admin/src/templates/index.html

```diff
@@ -15,7 +15,7 @@
         <div class="navigation">
           <ul>
             <li ng-class="{'active-nav': checkActive('display')}" mm-auth="can_configure"><a ui-sref="display.date-time" translate>admin.display</a></li>
-            <li ui-sref-active="active-nav" mm-auth="can_configure"><a ui-sref="users" translate>Users</a></li>
+            <li ng-class="{'active-nav': checkActive('users')}" mm-auth="can_configure"><a ui-sref="users" translate>Users</a></li>
```

Comment:
I don't think this change is needed if the state name is updated as per my previous suggestion.